testing/fceux/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Danct12 <danct12@disroot.org>
+# Maintainer: Danct12 <danct12@disroot.org>
+pkgname=fceux
+pkgver=2.2.3
+pkgrel=0
+pkgdesc="Fast and ultra-compatible NES/Famicom emulator"
+url="http://www.fceux.com"
+arch="all"
+license="GPL"
+depends="sdl gd gtk+2.0"
```

Comment:
there should be automatically found by the package manager by looking at the linked libraries

Reply:
So.. this depends line could be removed entirely?